CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Changelog
 
+## 12.4.0 (05/15/23)
+
+### Teleport Assist
+
+This release of Teleport introduces a preview release of Teleport Assist, a
+GPT-4 powered chat assistant that allows you to execute commands on servers
+connected to your Teleport cluster.
+
+It is available for all Teleport Cloud customers starting from 05/19/23.
```

Comment:
Not true, only the Team plan gets Assist for now.

Reply:
Yep, I will update.